data-reconciliation-app/docs/adr/03-reconciliation-logic.md

```diff
@@ -57,40 +57,38 @@ Each record will have all members' opinions for it. So for each record the recon
-Reconciliation logic is considered only if a specified number of opinions for a record is submitted. The `voting_threshold` determines if the record has received enough opinions to be reconciled. This will be configurable and can be set by members of the network, depending on the size of the network.
+Reconciliation logic is considered only if a specified number of users have submitted that record. The `voting_threshold` determines if the record has been submitted by enough users. This is configurable and can be set by members of the network, depending on the size of the network.
 
 ### For each record in the KV Storage
 
-First check to be made is whether the current User has submitted an opinion for this record. If not, this record can and will be skipped straightaway.
+The first thing to look for is whether the current User has submitted any value for this record. If not, this record will be skipped right away.
 
-If we have an opinion, then reconciliation can have 3 possible status:
+Otherwise, the reconciliation for that particular record can have three possible states:
 
-- `NOT_ENOUGH_DATA` : If number of opinions does not reach `voting_threshold`.
-- `LACK_OF_CONSENSUS`: If threshhold is met and data is not equal among all opinions.
-- `IN_CONSENSUS`: If threshold is met and data is equal among all opinions.
+- `NOT_ENOUGH_DATA` : If the count of users who submitted that key is less than the `voting_threshold`.
+- `LACK_OF_CONSENSUS`: If the threshold is met and at least one of the submitted values differs from all of the others.
+- `IN_CONSENSUS`: If the threshold is met and all the users have submitted the same value.
 
-**A single different record is already a reason for the result to be `LACK_OF_CONSENSUS`.**
-
-This status is the one that will be reported as the `group_status` field in the final Report.
+This state will come under the `group_status` field in the final Report.
 
 ### Sample Summary Result Schema and Report Mapping
 
 In addition to `group_status`, additional statistics will be retrieved for the record to be later consolidated in the Report:
 
-- Total number of opinions (this number helps reporting but shall not be public)
-- Number of unique opinions (count of different opinions for the record)
-- Number of members that agree with the current User
+- Total number of values submitted (useful for reporting but will not be made public)
+- Number of distinct values for that key (count of different opinions for the record)
+- Number of members that agree with the current user
 - If the member's opinion is in accordance with the majority or minority of votes
 
-Information will be consolidated in a Summary Result object that will be used by the Reporting API.
+Information will be consolidated into a Summary Result object that will be used by the Reporting API.
 The following proposed schema represents a reconciled record. The output Report will then be generated based on this data.
 
 ```json
 {
   "key": "<record key>",
```

Comment:
rename key to `lei`

rename value to `nace`

Also make comment, the key and value will match the schema defined

Reply:
Updated!